src/org/parosproxy/paros/extension/report/ReportGenerator.java

```diff
@@ -126,7 +127,7 @@ public static File stringToHtml(String inxml, String infilexsl, String outfilena
 					bw = new BufferedWriter(new FileWriter(outfilename + "-orig.xml"));
 					bw.write(inxml);
 				} catch (IOException e2) {
-					logger.error(e.getMessage(), e);
+					logger.error(Constant.messages.getString("report.write.diskspace.error.log.message"), e);

```

Comment:
I don't think the log message should be internationalised, that makes it harder to (immediately) understand the logs (although in this case it might not be that bad).